mmdet/models/dense_heads/anchor_head.py

```diff
@@ -8,10 +8,11 @@
                         multiclass_nms, unmap)
 from ..builder import HEADS, build_loss
 from .base_dense_head import BaseDenseHead
+from .dense_test_mixins import BBoxTestMixin
 
 
 @HEADS.register_module()
-class AnchorHead(BaseDenseHead):
+class AnchorHead(BaseDenseHead, BBoxTestMixin):
```

Comment:
RPNHead has a mixin `RPNTestMixin`, can these two mixins somehow be merged to avoid confusion?

Reply:
Simple merge of `dense_test_mixins.py` and `rpn_test_mixin.py` causes another confusion, because `dense_test_mixins.py` focuses on TTA, and doesn't have `simple_test`.

This issue comes from the following inconsistencies.

- `roi_heads` have `simple_test`
- `RPNHead` and `GARPNHead` have `simple_test_rpn`
- `dense_heads` don't have `simple_test`

I think the inconsistencies should be addressed by a PR for refactoring, not by this PR for adding a feature.

Reply:
Agree. A later refactoring will be proposed.